utils/submission.js

```diff
@@ -233,18 +233,20 @@ const getParticipants = async (req, res, authObj) => {
         }
     }
     else if(req.query.type === 'filter') {
+        const filterDeprecationWarning = "IMPORTANT: this type == 'filter' API is deprecated. Please use the new 'getFilteredParticipants' API. " + 
+            "Ex: .../getFilteredParticipants?firstName=John&lastName=Doe | API notes & documentation: https://github.com/episphere/connect/issues/817#issuecomment-1883893946";
```

Comment:
Very nice touch. As a side note, if it hasn't happened already, let's make sure Davin and Michelle know about this so we can get official communication out to the sites.

Reply:
Thanks, I'll merge and reach out about it now.